packages/gestalt/src/Avatar/Avatar.js

```diff
@@ -20,7 +20,7 @@ const Square = (props: *) => (
 );
 
 const DefaultAvatar = ({ name }: { name: string }) => {
-  const firstInitial = [...name][0].toUpperCase();
+  const firstInitial = name ? [...name][0].toUpperCase() : '';
```

Comment:
Since react renders empty strings into the DOM should that be a null or undefined?

Reply:
Both seem equivalent from an end-user perspective. What's wrong with empty strings in the DOM?

Reply:
Feels dirty, Airbnb has a rule of always explicitly checking against an empty string or a 0 instead of relying on the implicit Boolean conversion so that you don’t get empty strings and 0s in the DOM when you do a conditional render. This sort of feels the same lines. Just thought we should make gestalt as clean as possible for all users

Reply:
It sounds like you're talking about 2 different issues - (1) the implicit boolean conversion and (2) putting an empty string in the DOM. No opinion on either in general, but I think in this specific case they're both safe. If the reviewer has an opinion I'm happy to update.

Reply:
Agreed - leaving it up to @christianvuerings

Reply:
@bradencanderson @pfarejowicz I'm ok with the empty string in this case. However, we should conditionally render the SVG (within the `Square`) below so we don't render DOM that's not required.

```jsx
<Square color="gray" shape="circle">
  {firstInitial && <svg
```